app/products/playbooks/screens/playbook_run/playbook_run.tsx

```diff
@@ -249,25 +250,24 @@ export default function PlaybookRun({
         );
     }, [handleSelectOwner, intl, owner, participants, playbookRun?.name, theme]);
 
-    // this will be back once there is a rename function on the server side
-    // const handleRename = useCallback(async (newName: string) => {
-    //     if (!playbookRun) {
-    //         return;
-    //     }
+    const handleRename = useCallback(async (newName: string) => {
```

Comment:
Any strong reason to have the `handleRename` here, instead of having it defined on the rename screen?